llarp/service/endpoint.cpp

```diff
@@ -345,6 +345,25 @@ namespace llarp
       itr->second.lastUsed = Now();
     }
 
+    size_t
+    Endpoint::RemoveAllConvoTagsFor(service::Address remote)
+    {
+      size_t removed = 0;
+      auto& sessions = Sessions();
+      auto itr = sessions.begin();
+      while (itr != sessions.end())
```

Comment:
This section could use std::find_if, yes?

Reply:
could do `for (auto itr = sessions.find_if(...); itr != sessions.end(); itr = sessions.find_if(...))` I think

Reply:
not sure if that's better/more clear though

Reply:
probably not worth the effort to change, not clearly better and possibly worse.